nodejs/dist/uws.js

```diff
@@ -289,6 +289,20 @@ class Server extends EventEmitter {
         this._upgradeListener = null;
         this._noDelay = options.noDelay === undefined ? true : options.noDelay;
         this._lastUpgradeListener = true;
+        this._abortConnection = (socket, code, name) => {
+          try {
+            const response = [
+              'HTTP/1.1 ' + code + ' ' + name,
+              'Content-Type: text/html'
+            ];
+            socket.write(response.concat('', '').join('\r\n'));
+          }
+          catch (e) { /* ignore errors - we've aborted this connection */ }
+          finally {
+            // ensure that an early aborted connection is shut down completely
+            try { socket.destroy(); } catch (e) {}
```

Comment:
This calls close(fd), so it won't guarantee what you send will be delivered as opposed to end which calls shutdown.